contrib/scripts/ablation.sh

```diff
@@ -3,22 +3,22 @@
 source activate seismic-interpretation
 
 # Patch_Size 100: Patch vs Section Depth
-python scripts/prepare_dutchf3.py split_train_val patch --data_dir=/mnt/dutch --stride=50 --patch_size=100
+python scripts/prepare_dutchf3.py split_train_val patch --data_dir=/mnt/dutch --stride=50 --patch_size=100 --split_direction=crossline
```

Comment:
Again, this flag is not needed, right? Same for the other lines below.

Reply:
we discussed this offline and agreed to add a "both" option to the split_direction